src/settings/Settings.tsx

```diff
@@ -230,6 +230,20 @@ export const SETTINGS: { [setting: string]: ISetting } = {
             requiresRefresh: true,
         },
     },
+    "feature_notifications_panel": {
+        isFeature: true,
+        labsGroup: LabGroup.Messaging,
+        displayName: _td("Notifications panel"),
+        supportedLevels: LEVELS_FEATURE,
+        default: false,
+        // Reload to ensure that the left panel etc. get remounted
+        controller: new ReloadOnChangeController(),
+        betaInfo: {
+            title: _td("Notifications panel"),
+            caption: () => <p>{_t("A list of all events notifying you. Only works with unencrypted rooms")}</p>,
```

Comment:
I don't think it's desirable to be overly specific here. I'd be open to hearing alternative wording though.

This is how everyone internally at Element has described to me how this feature works. I'm trying to avoid being technically overly accurate to not overwhelm a more average user that might not grasp all the tiny distinctions between the concepts discussed [here](https://github.com/vector-im/element-web/pull/25924#discussion_r1284276394)

Reply:
Looping @daniellekirkwood and @nadonomy here

Reply:
Yeah, my understanding is that it only works with Unencrypted rooms so this feels accurate.

Reply:
The `/notifications` API in the spec does not discriminate whether a room is encrypted or not. The actual matching is for events which match set_tweak `highlight` in your push rules. So if you disable `Noisy` notifications in unencrypted rooms it won't even include those.